src/appkit.cs

```diff
@@ -18492,6 +18771,14 @@ partial interface NSTextInputContext {
 		[Static]
 		[Export ("localizedNameForInputSource:")]
 		string LocalizedNameForInputSource (string inputSourceIdentifier);
+
+		[Mac (14, 0)]
+		[Export ("textInputClientWillStartScrollingOrZooming")]
+		void TextInputClientWillStartScrollingOrZooming ();
```

Comment:
I know we typically look to the Swift naming convention but in this case its not any better, but would this possibly be a better name to expose?
```suggestion
		void StartScrollingOrZooming ();
```

Reply:
Leaving as it is.